tcav/results.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto2";
+
+package tcav.tcav;
+
+
+message Result {
```

Comment:
add some comments throughout explaining the messages and any non-obvious vars

Reply:
done